release.py

```diff
@@ -475,7 +475,7 @@ def command_release_announcement_email():
 rc = raw_input("Release candidate number: ")
 
 dev_branch = '.'.join(release_version_parts[:2])
-docs_release_version = docs_version(release_version[:2])
+docs_release_version = docs_version(release_version)
```

Comment:
Comparing with https://github.com/apache/kafka/pull/6317/files#diff-97f2783dc2d2b4c99fad05f863e53686R216

It seems we need to pass in the full version string, eg, `2.2.0` -- I run the script and in computed `2` as result instead of `22`

Reply:
yeah, seems valid. we already chop off whatever we don't need in `docs_version`, I guess just leftover from the `0.` days...